components/Buttons/UserButton.vue

```diff
@@ -8,7 +8,7 @@
       >
         <user-image />
         <h1 class="font-weight-light pb-1 text-truncate">
-          {{ $auth.user.Name }}
+          {{ userName }}
```

Comment:
The component is entirely useless and shouldn't even be shown if there is no user, so I don't really see the reason for this.

Reply:
It's also usually nicer to do this in the template instead of creating a computed property just for a simple check like this.

Reply:
@mrtimscampi it's the one causing issues in fact, as the home header welcome one it's not active most of the time.

I can move the checks there, though, although as you mention, they are no longer *that* necessary as the logout process is fired earlier.

That's why I think it's a good idea to keep them, but in the template, just for safety.

Also, I noticed (specially when working with home header) that there are some times when *if* checks happen with some delay, so computed properties take place before, causing issues.